packages/esm-patient-conditions-app/src/conditions/conditions-form.component.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { SyntheticEvent } from 'react';
 import dayjs from 'dayjs';
-import debounce from 'lodash-es/debounce';
+import { debounce } from 'lodash-es';
```

Comment:
Why this change? Please keep the submodule imports. Otherwise you have the full lodash in here.

Reply:
Fixed in the latest commit.